spec/requests/providers/check_merits_answers_controller_spec.rb

```diff
@@ -1,7 +1,7 @@
 require "rails_helper"
 require "sidekiq/testing"
 
-RSpec.describe "check merits answers requests" do
+RSpec.describe Providers::CheckMeritsAnswersController do
```

Comment:
This seems new, and not immediately related to the change, is this a CYC?

Reply:
Yes, on most other controller tests, we use the class name in the describe block, also the file name should match the pattern, `controller-name_controller_spec.rb`

Bonus for me -- it makes it easier for rubymine to jump between object and tests!